crypto/keyring/keyring.go

```diff
@@ -426,6 +429,31 @@ func (ks keystore) DeleteByAddress(address sdk.Address) error {
 	return nil
 }
 
+func (ks keystore) Rename(oldName, newName string) error {
+	_, err := ks.Key(newName)
+	if err == nil {
+		return errors.New(fmt.Sprintf("rename failed: %s already exists in the keyring", newName))
+	}
+
+	passPhrase := "temp"
```

Comment:
nit: this should be a private const

Reply:
yup that makes much more sense - [moved it](https://github.com/cosmos/cosmos-sdk/pull/9601/commits/2c7395bddf4e833e2c6b9ee0f8193d7cb76cc5f8) to the other private const vars